[2021-04-23 20:38:07,689] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-23T17:00:00+00:00 [queued]>
[2021-04-23 20:38:07,783] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-23T17:00:00+00:00 [queued]>
[2021-04-23 20:38:07,788] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-23 20:38:07,790] {taskinstance.py:1043} INFO - Starting attempt 2 of 2
[2021-04-23 20:38:07,792] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-23 20:38:07,816] {taskinstance.py:1063} INFO - Executing <Task(DataprocCreateClusterOperator): create_cluster_dataproc> on 2021-04-23T17:00:00+00:00
[2021-04-23 20:38:07,826] {standard_task_runner.py:52} INFO - Started process 504 to run task
[2021-04-23 20:38:07,836] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'recent_played_ingestion', 'create_cluster_dataproc', '2021-04-23T17:00:00+00:00', '--job-id', '36', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/recent_played_ingestion.py', '--cfg-path', '/tmp/tmpuwlc6qby', '--error-file', '/tmp/tmp1fmhead8']
[2021-04-23 20:38:07,851] {standard_task_runner.py:77} INFO - Job 36: Subtask create_cluster_dataproc
[2021-04-23 20:38:08,047] {logging_mixin.py:104} INFO - Running <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-23T17:00:00+00:00 [running]> on host 760ffd3d790b
[2021-04-23 20:38:08,229] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=recent_played_ingestion
AIRFLOW_CTX_TASK_ID=create_cluster_dataproc
AIRFLOW_CTX_EXECUTION_DATE=2021-04-23T17:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-23T17:00:00+00:00
[2021-04-23 20:38:08,240] {dataproc.py:603} INFO - Creating cluster: ephemeral-recent-played-spark-cluster-20210423
[2021-04-23 20:42:19,633] {dataproc.py:547} INFO - Cluster created.
[2021-04-23 20:42:28,736] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=recent_played_ingestion, task_id=create_cluster_dataproc, execution_date=20210423T170000, start_date=20210423T203807, end_date=20210423T204228
[2021-04-23 20:42:32,270] {local_task_job.py:146} INFO - Task exited with return code 1
